app/controllers/api_controller.rb

```diff
@@ -1,3 +1,8 @@
+#
+# Initializing REST API environment, called once @ startup
+#
+Api::Initializer.new.go
```

Comment:
Bump for @Fryguy @chrisarcand @bdunne. Note: I believe the above preserves the current behavior, but I'd love for this to move to a sturdier solution